panacea/grpc_client.go

```diff
@@ -84,3 +133,109 @@ func (c *GrpcClient) GetDeal(id string) (markettypes.Deal, error) {
 
 	return *response.GetDeal(), nil
 }
+
+// RegisterDataValidator registers data validator on blockchain
+func (c *GrpcClient) RegisterDataValidator(address, endpoint string) error {
```

Comment:
It's good to go with tx signing/broadcasting logic in this function, even though we can generalize it later. Thank you 👍